postgres/datadog_checks/postgres/util.py

```diff
@@ -340,6 +340,20 @@ def payload_pg_version(version):
     ],
 }
 
+QUERY_PG_REPLICATION_INACTIVE_SLOTS = {
+    'name': 'pg_replication_inactive_slots',
+    'query': """
+    SELECT
+        COUNT(*)
+        FROM pg_replication_slots
```

Comment:
There is already a full table scan on this table. Can the same be accomplished with editing the existing `QUERY_PG_REPLICATION_SLOTS`? Something like this:


```diff
QUERY_PG_REPLICATION_SLOTS = {
    'name': 'pg_replication_slots',
    'query': """
    SELECT
        slot_name,
        slot_type,
        CASE WHEN temporary THEN 'temporary' ELSE 'permanent' END,
        CASE WHEN active THEN 'active' ELSE 'inactive' END,
        CASE WHEN xmin IS NULL THEN NULL ELSE age(xmin) END,
        pg_wal_lsn_diff(
        CASE WHEN pg_is_in_recovery() THEN pg_last_wal_receive_lsn() ELSE pg_current_wal_lsn() END, restart_lsn),
        pg_wal_lsn_diff(
        CASE WHEN pg_is_in_recovery() THEN pg_last_wal_receive_lsn() ELSE pg_current_wal_lsn() END, confirmed_flush_lsn),
+       1
    FROM pg_replication_slots;
    """.strip(),
    'columns': [
        {'name': 'slot_name', 'type': 'tag'},
        {'name': 'slot_type', 'type': 'tag'},
        {'name': 'slot_persistence', 'type': 'tag'},
        {'name': 'slot_state', 'type': 'tag'},
        {'name': 'postgresql.replication_slot.xmin_age', 'type': 'gauge'},
        {'name': 'postgresql.replication_slot.restart_delay_bytes', 'type': 'gauge'},
        {'name': 'postgresql.replication_slot.confirmed_flush_delay_bytes', 'type': 'gauge'},
+.      {'name': 'postgresql.replication_slot.slots', 'type': 'count'},
    ],
}
```

with Datadog query something like:

```
sum:postgresql.replication_slot.slot_count{slot_state:active} by {*}
```

IDK if this will work exactly (untested), but if you can get it to work, it seems preferable to a very narrow query and very narrow metric. e.g what if I want to see the "active" slots?